cypress/integration/aragon.spec.js

```diff
@@ -11,144 +18,368 @@ context('Aragon', () => {
+        // click on the first vote card
+        cy.wait(5000).getWithinIframe('div.XwjlG').first().click().wait(1500)
+        .getWithinIframe('Button:contains("Vote")').click()
+        // click on the slider at a point where a non-zero value will be set
+        .getWithinIframe('div.jnspDJ').trigger('mousedown', { which: 1, x: 500, y: 5}).trigger('mouseup').wait(500)
+        cy.getWithinIframe('button:contains("Submit Vote")').click().wait(1000)
 
-  it('.find() - get descendant DOM elements of the selector', () => {
-    // https://on.cypress.io/find
-    cy.get('.traversal-pagination')
-      .find('li').find('a')
-      .should('have.length', 7)
-  })
+        cy.contains('button','Create transaction').click()
+        cy.contains('button','Close').click().wait(5000)
 
-  it('.first() - get first DOM element', () => {
-    // https://on.cypress.io/first
-    cy.get('.traversal-table td')
-      .first().should('contain', '1')
-  })
+        /* left as a potential reference for validation
+        .get('iframe').iframe().wait(1000).find('div').each((el) => {
+          if (/VoteStatus__StatusLabel/.test(el[0].className)) {
+            console.log('++ found sidepanel', el[0])
+            cy.wrap(el[0]).within(($el) => {
+  
+              cy.find('span').each((el) => {
+                if (/VoteStatus__StatusLabel/.test(el[0].className)) {
+                  console.log('+!!!+', el[0])
+                }
+              })
+              
+            })
+          }
+        })
+        .get('iframe').iframe().find('input').each((el) => {
+          if (/TextInput/.test(el[0].className)) {
+            console.log('--', el[0].className)
+            el[0].click()
+          }
+        })
+        */
+      })
 
-  it('.last() - get last DOM element', () => {
-    // https://on.cypress.io/last
-    cy.get('.traversal-buttons .btn')
-      .last().should('contain', 'Submit')
-  })
+      it('executes the vote', () => {
+        // click on the most recent vote card
+        //cy.getWithinIframe('button:contains("Back")').click()
+        cy.getWithinIframe('div.XwjlG').first().click().wait(60000)
+        cy.getWithinIframe('button:contains("Execute Vote")').click()
 
-  it('.next() - get next sibling DOM element', () => {
-    // https://on.cypress.io/next
-    cy.get('.traversal-ul')
-      .contains('apples').next().should('contain', 'oranges')
+        cy.contains('button','Create transaction').click()
+      cy.contains('button','Close').click().wait(1000)
+      })
+    })
   })
 
-  it('.nextAll() - get all next sibling DOM elements', () => {
-    // https://on.cypress.io/nextall
-    cy.get('.traversal-next-all')
-      .contains('oranges')
-      .nextAll().should('have.length', 3)
-  })
+  context('Projects', () => {
+    let unfetch
+    before('logs into Projects', () => {
+      cy.contains('button','Projects',{ timeout: 150000 }).click().wait(10000)
+      cy.getWithinIframe('button:contains("Sign In")').click()
```

Comment:
We don't need to click this button to spoof the sign in flow. Since we can't interact with the popup anyhow, I don't think it's useful.

Reply:
We click the button to create the `message` event listener in our app code. Once that listener is active. we post our own message to the app window that gets us logged in.

Reply:
Ah, this will change once we finally get #1300 merged!

Reply:
clicking the button is still the easiest way to generate a popup handle and inject the message so I've kept this in